integration/spark/src/main/scala/org/apache/carbondata/view/MVHelper.scala

```diff
@@ -291,6 +307,23 @@ object MVHelper {
     fieldsMap
   }
 
+  private def findDuplicateColumns(fieldColumnsMap: util.HashMap[String, util.ArrayList[String]],
```

Comment:
please handle the case of one column with direct name and one with alias.

Reply:
Handled